activators/dialogflow/src/main/kotlin/com/justai/jaicf/activator/dialogflow/DialogflowIntentActivator.kt

```diff
@@ -44,9 +39,47 @@ class DialogflowIntentActivator(
         return intent?.let { listOf(DialogflowActivatorContext(intent, qr)) } ?: emptyList()
     }
 
-    class Factory(private val connector: DialogflowConnector): ActivatorFactory {
+    override fun fillSlots(
+        request: BotRequest,
+        reactions: Reactions,
+        botContext: BotContext,
+        activatorContext: ActivatorContext?,
+        slotReactor: SlotReactor?
+    ): SlotFillingResult {
+        var context = activatorContext?.dialogflow?.apply {
+            if (queryResult.allRequiredParamsPresent) {
+                return SlotFillingSkipped
+            }
+        }
+        if (context == null) {
+            context = recogniseIntent(botContext, request).firstOrNull()
+        }
+        return when {
+            context == null -> SlotFillingInterrupted
+            context.queryResult.allRequiredParamsPresent -> SlotFillingFinished(context)
```

Comment:
Shouldn't we respond with Dialogflow prompt message? The same for `SlotFillingSkipped` case.

-> Hello
<- Hi, what is your name?
-> John
<- Nice to meet you, John! (will not be displayed)